proto/reservation/v1/reservation.proto

```
@@ -50,7 +50,7 @@ message ReserveRequest {
   string request_id = 2;
   // amount is a decimal serialized as a string to avoid float rounding.
   string amount = 3;
-  string currency = 4;
+  string asset = 4;
```

Buf (1.72.0)

[error] Field "4" with name "asset" on message "ReserveRequest" changed option "json_name" from "currency" to "asset".

(FIELD_SAME_JSON_NAME)